src/odemis/gui/cont/tabs/correlation_tab.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+
+"""
+@author: Rinze de Laat, Éric Piel, Philip Winkler, Victoria Mavrikopoulou,
+         Anders Muskens, Bassim Lazem, Patrick Cleeve
+
+Copyright © 2012-2022 Rinze de Laat, Éric Piel, Delmic
+
+Handles the switch of the content of the main GUI tabs.
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License version 2 as published by the Free
+Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+"""
+
+import collections
+import logging
+import os.path
+from typing import List
+
+import wx
+# IMPORTANT: wx.html needs to be imported for the HTMLWindow defined in the XRC
+# file to be correctly identified. See: http://trac.wxwidgets.org/ticket/3626
+# This is not related to any particular wxPython version and is most likely permanent.
+import wx.html
+
+from odemis.gui import conf
+
+from odemis import dataio
+from odemis import model
+import odemis.gui
+import odemis.gui.cont.export as exportcont
+import odemis.gui.cont.streams as streamcont
+import odemis.gui.cont.views as viewcont
+import odemis.gui.model as guimod
+import odemis.gui.util as guiutil
+from odemis.acq.stream import OpticalStream, EMStream, StaticStream
+from odemis.gui.cont.correlation import CorrelationController
+from odemis.gui.model import TOOL_ACT_ZOOM_FIT
+from odemis.gui.util import call_in_wx_main
+from odemis.gui.cont.tabs import TOOL_ORDER
```

Comment:
You really want to avoid importing `.tabs`, as `.tabs` imports this module. It's a cyclic import. Python sometimes manages to handle that (and that's the probably the case here), but it's very brittle. Just moving the import order can break it and cause a lot of hair pulling. 
=> no constants used by these sub-modules should be directly defined in `__init__.py`.